02-generalize/02.rb

```diff
@@ -0,0 +1,40 @@
+# To make a empty square
+class Square
+  def initialize(side,display_char)
+    @side = side
+    @display_char = display_char
```

Comment:
Consider using a simpler name that `display_char`; the program is essentially creating a pattern using a symbol. This may not even be for displaying, so try not to use specific names for variables.